packages/core/src/messages/responses/topic-item.ts

```diff
@@ -14,10 +14,16 @@ export interface TopicItemOptions {
 export class TopicItem {
   private readonly _value: string | Uint8Array;
   private readonly _tokenId?: string;
+  private readonly _sequenceNumber: number;
 
-  constructor(_value: string | Uint8Array, options?: TopicItemOptions) {
+  constructor(
```

Comment:
This is technically a breaking change to this constructor but I don't expect anyone external to be constructing these, so I think what you've done here is probably the best way to go.